server/utils/socket_config.py

```diff
@@ -2,17 +2,20 @@
 from flask_socketio import send, emit, join_room, leave_room, SocketIO
 
 
-@socketio.on('connect')
+@socketio.on("connect")
 def connected():
-    emit('message', "You have connected")
+    emit("message", "You have connected")
 
 
-@socketio.on('subscribeToNotifications')
+@socketio.on("review_message")
+def handleMessage(data):
+    socketio.emit(data["review_id"], {"message_id": data["message_id"]})
```

Comment:
Add a bit of logic to only emit an event to the person that is receiving the message